pkg/doc/verifiable/credential.go

```diff
@@ -1197,11 +1209,67 @@ func getSchemaLoader(schemas []TypedID, opts *credentialOpts) (gojsonschema.JSON
 	}
 
 	// If no custom schema is chosen, use default one
-	return defaultSchemaLoader(), nil
+	return defaultSchemaLoaderWithOpts(opts), nil
+}
+
+type defaultSchemaOpts struct {
+	disabledChecks []string
+}
+
+// DefaultSchemaOpt is create default schema options.
+type DefaultSchemaOpt func(*defaultSchemaOpts)
+
+// WithDisableRequiredField disabled check of required field in default schema.
+func WithDisableRequiredField(fieldName string) DefaultSchemaOpt {
+	return func(opts *defaultSchemaOpts) {
+		opts.disabledChecks = append(opts.disabledChecks, schemaPropertyIssuanceDate)
+	}
+}
+
+// CreateDefaultSchema creates default schema with the option to disable the check of specific properties.
+func CreateDefaultSchema(opts ...DefaultSchemaOpt) string {
```

Comment:
I am bit unsure about Default name here - default is already configured in the code base, this option is used by clients to override the those options. I feel, following names would be more concise
- CreateDefaultSchema - > JSONSchemaLoader : similar to JSON-LD schema loader https://github.com/hyperledger/aries-framework-go/blob/aa5f611f87a0b3f6c335ee8d6f2e699ac6509428/pkg/doc/verifiable/credential.go#L301

- DefaultSchemaOpt -> SchemaOpt : since we are overriding schema

https://github.com/hyperledger/aries-framework-go/blob/aa5f611f87a0b3f6c335ee8d6f2e699ac6509428/pkg/doc/verifiable/credential.go#L301